### PR TITLE
Boot2Docker Cleanup 

### PR DESCRIPTION
**Pull request by moxiegirl**
- Remove references to sudo in basics.md; see sudo instructions top of file
- Removing references to Boot2Docker replacing with Docker Machine
- Removing sudo warnings in instances where appropriate (no sudo in file)

Signed-off-by: Mary Anthony mary@docker.com

### CODE REVIEW

**Comment by moxiegirl** [comment]
ping @docker/docs-owners

**Comment by aidanhs** [comment]
I assume by sudo instructions you're referring to

>  Additionally, depending on your Docker system configuration, you may be required
 to preface each `docker` command with `sudo`. To avoid having to use `sudo` with
 the `docker` command, your system administrator can create a Unix group called
 `docker` and add users to it.

?

But these instructions only relate to running the client, not the daemon - the daemon must always be run as root, even if you're in the docker group.

File: docs/README.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ own.
 	container with this image.
 
 	The container exposes port 8000 on the localhost so that you can connect and
-	see your changes. If you are running Boot2Docker, use the `boot2docker ip`
+	see your changes. If you are running Docker Machine, use the `docker-machine ip`
```
Comment (thaJeztah) [inline docs/README.md]: perhaps *using* docker machine (in stead of *running*)

File: docs/project/test-and-docs.md [hunk 2]
```diff
@@ -253,9 +253,9 @@ can browse the docs.
 
 4. Enter the URL in your browser.
 
-    If you are running Boot2Docker, replace the default localhost address
+    If you are running Docker Machine, replace the default localhost address
```
Comment (thaJeztah) [inline docs/project/test-and-docs.md]: s/running/using/

**Comment by thaJeztah** [comment]
Thanks @moxiegirl, some really minor nits.

@aidanhs is right though (for running the daemon), perhaps you can add something to that note, or to the section about running the daemon?

**Comment by moxiegirl** [comment]
@aidanhs @thaJeztah I've adjusted the note to refer to the commands on the page.  There are no examples of running the daemon here.  Also added the sudo back into the daemon runs in `basics.md`

**Comment by thaJeztah** [comment]
> There are no examples of running the daemon here. 

oh! you're right :-)


LGTM thanks!

**Comment by thaJeztah** [comment]
ping @calavera for cherry pick

**Comment by calavera** [comment]
cherry picked into #15091

**Pull request merged**